casper/src/main/scala/coop/rchain/casper/engine/CasperLaunch.scala

```diff
@@ -123,18 +115,15 @@ object CasperLaunch {
                 init.conf.approveGenesisDuration,
                 init.conf.approveGenesisInterval
               )
-      // TODO OMG Fix, use Concurrent+!11
-      _ <- Sync[F].delay {
-            val _ = toTask(
-              GenesisCeremonyMaster
-                .approveBlockInterval[F](
-                  init.conf.approveGenesisInterval,
-                  init.conf.shardId,
-                  validatorId
-                )
-            ).executeWithOptions(TaskContrib.enableTracing(tracing)).forkAndForget.runToFuture
-            ().pure[F]
-          }
+      // TODO track fiber
+      _ <- Concurrent[F].start(
```

Comment:
Last time I tried to do that, there have been some test failures. Did you observe any and/or fixed the bug(s)? They could've been fixed in the meantime ofc.

Reply:
I have not seen any unit test fail due to this change. but believe me, I'm looking.